components/meta-packages/history/history

```diff
@@ -534,12 +534,15 @@ desktop/virt-manager@0.6.1,5.11-2015.0.2.0
 developer/build/automake-19@1.9.6,5.11-5.12.0.0.0.16.0
 developer/build/cbe@1.8.1,5.11-2017.0.0.1
 developer/clang-3.3@3.3,5.11-2015.0.1.2
+developer/clang-34@3.4.2-2017.0.0.5
```

Comment:
please, bump latest number (2017.0.0.5 => 2017.0.0.6) and so on to avoid dependency on timestamp